lib/js/_validate/_check-package-structure.js

```diff
@@ -116,34 +134,37 @@ function isFileType(item) {
 
 /**
  * Filter glob results
- * - Remove directory string
  * - Remove paths matching .gitignore'd files
  * - Remove dotfiles
  * @private
  * @function getFilteredResults
- * @param {String} directory path to files
  * @param {Array} files file list to filter
  * @return {Array}
  */
-function getFilteredResults(directory, files) {
+function getFilteredResults(files) {
```

Comment:
Bit ambiguous..? maybe `removeIgnoredFiles`? or split into two functions, `removeDotFiles` and `removeGitIgnoredFiles` ?

Reply:
It's explained in the function comment, but can rename it. not sure splitting it out serves to make it any clearer.